pages/instructorAssessmentInstances/instructorAssessmentInstances.ejs

```diff
@@ -270,8 +293,8 @@
                   widgets: ["uitheme"],
                   headers: {
                     <% if (assessment.group_work) { %>
-                      7: {sorter: false},
                       8: {sorter: false},
+                      9: {sorter: false},
```

Comment:
Does the `else` block need to be bumped to 9 and 10? Otherwise it looks like we can just ditch the `if`/`else`.